core/src/main/java/org/apache/shiro/session/Session.java

```diff
@@ -149,7 +149,7 @@
      * It is common for a {@code Subject} implementation to retain authentication state in the
      * {@code Session}.  If the session
      * is explicitly stopped by application code by calling this method directly, it could clear out any
-     * authentication state that might exist, thereby effectively &quot;unauthenticating&quot; the {@code Subject}.
+     * authentication state that might exist, thereby effectively removing the &quot;authenticated&quot; state of the {@code Subject}.
```

Comment:
Not 100% sure about this one. I know "to unauthenticate" is not a common word, but it is easier to understand than "removing the >authenticated<-state".

Reply:
_unauthenticate_ really isn't a word. Conceptually the word would be _deauthenticate_ (but even that isn't in Google Chrome's dictionary -- nor is it in the dictionary I'm using, although I'd be ok w/ it).

Reply:
Maybe @bdemers can think of something else?